dev.txt

```diff
@@ -0,0 +1,4 @@
+./datadog-checks-base 
```

Comment:
what's `dev.txt`?

Reply:
🤷‍♂️

Reply:
Probably an artifact that I forgot to delete while playing with the new test system. I'll remove it